src/client/pages/PageRoutes/hooks/useRoutes.tsx

```diff
@@ -4,6 +4,8 @@ import { createRoutesFromElements, Navigate, Route } from 'react-router-dom'
 import { RegionCode } from 'meta/area'
 import { Routes } from 'meta/routes/routes'
 
+import Kiosk from 'client/components/Kiosk/Kiosk'
+import KioskPageLayout from 'client/components/Kiosk/PageLayout'
```

Comment:
@minotogna I added the Kiosk related components at `client/components/Kiosk/` for now. Let me know if you think those should go in a different folder.